react/components/atoms/Input/input.stories.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { storiesOf } from "@storybook/react-native";
+import { withKnobs, bool, text } from "@storybook/addon-knobs";
+
+import { InputRipe } from "./input";
+
+storiesOf("Components/Atoms/Input", module)
+    .addDecorator(withKnobs)
+    .add("Input", () => {
+        const header = text("Header", undefined);

```

Comment:
Add a initial value so that we can see the new features right when we enter the component